src/components/app/routes/data/utils.js

```diff
@@ -179,21 +181,21 @@ export function redirectToRemoveTrailingSlash(requestUrl) {
   throw redirect(requestUrl.pathname.slice(0, -1));
 }
 
+configureLogging(NewRelicLoggingService, {
+  config: getConfig(),
+});
+configureAuth(AxiosJwtAuthService, {
+  loggingService: getLoggingService(),
+  config: getConfig(),
+});
```

Comment:
[inform] Ended up realizing there is a downside to calling `configure*` within `ensureAuthenticatedUser` within the actual app experience; I was running into seemingly race conditions where the `authenticatedUser` would become unhydrated during a redirect from license activation route to the dashboard page.